src/components/dnspolicy/LoadBalancingField.tsx

```diff
@@ -31,7 +31,7 @@ const LoadBalancingField: React.FC<LoadBalancingProps> = ({ loadBalancing, onCha
           }
           isRequired
           type="number"
-          placeholder={t("0")}
```

Comment:
ó

Reply:
ah yes zero in non english languages qui qui